### PR TITLE
Sparrow dom/growth update

### PR DESCRIPTION
**Pull request by sparrowDom**
### Description:
Enables translations for Growth campaign short names and long names that are inserted into postgres. 
This is also added as a comment: 
When new growth campaign is added the short/long name translations need to be added to Growth translation enum file (`experimental/origin-dapp2/src/constants/Growth$FbtEnum.js`)

### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [ ] Wrap any displayed ETH addresses with [`formattedAddress`](https://github.com/OriginProtocol/origin/blob/master/origin-dapp/src/utils/user.js#L15-L17)
- [ ] Wrap any new text/strings for translation
- [ ] Run `npm run translations` if there are any changes to translated strings
- [ ] Map any new environment variables with a default value in the Webpack config
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/origin-docs)

### CODE REVIEW

File: origin-growth/migrations/20190226142347-update-growth-campaign.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
+'use strict'
```
Comment (franckc) [inline origin-growth/migrations/20190226142347-update-growth-campaign.js]: No need to create a new migration file. We haven't run those growth migrations yet on staging/prod.
So feel free to just modify the existing migration file that creates that table.

File: experimental/origin-dapp2/src/pages/App.js [hunk 2]
```diff
@@ -66,7 +66,8 @@ class App extends Component {
     const { creatorConfig } = this.props
     applyConfiguration(creatorConfig)
     const shouldRenderNavbar = this.props.location.pathname !== '/welcome'
-    const enableGrowth = process.env.ENABLE_GROWTH === 'true'
+    //const enableGrowth = process.env.ENABLE_GROWTH === 'true'
```
Comment (franckc) [inline experimental/origin-dapp2/src/pages/App.js]: remove ?

File: experimental/origin-dapp2/src/pages/App.js [hunk 3]
```diff
@@ -66,7 +66,8 @@ class App extends Component {
...
+    const enableGrowth = true
```
[the remark this answers is not in the document]
Comment (sparrowDom) [reply experimental/origin-dapp2/src/pages/App.js]: yup removed thanks

File: experimental/origin-dapp2/src/pages/App.js (hunk 2 above)
Comment (sparrowDom) [reply experimental/origin-dapp2/src/pages/App.js]: yup removed thanks

File: origin-growth/migrations/20190226142347-update-growth-campaign.js (hunk 1 above)
Comment (sparrowDom) [reply origin-growth/migrations/20190226142347-update-growth-campaign.js]: aha, tnx updating the initial migration

**Pull request merged**